mod_reforged/hooks/skills/special/double_grip.nut

```diff
@@ -294,9 +299,9 @@
 			}
 		}
 
-		if (_skill.getID() != "actives.puncture" && this.m.CurrWeaponType == ::Const.Items.WeaponType.Dagger)
+		if (_skill.getID() == "actives.puncture" && this.m.CurrWeaponType == ::Const.Items.WeaponType.Dagger)
```

Comment:
Just like in vanilla we now have to re-add some check like `canDoubleGrip` here.

Reply:
That check is already there due to the `this.m.CurrWeaponType` thing because that is only set to Dagger during `onUpdate` if `canDoubleGrip`.